docs/content/en/docs/workflows/dev.md

```diff
@@ -4,4 +4,44 @@ linkTitle: "Continuous development"
 weight: 1
 ---
 
-{{< todo 3074>}} dev page
\ No newline at end of file
+## `skaffold dev`
+
+`skaffold dev` enables continuous local development on an application.
+While in `dev` mode, Skaffold will watch an application's source files, and when it detects changes,
```

Comment:
```suggestion
While in `dev` mode, Skaffold will watch an application's source files, and when it detects changes, it
```

Reply:
I actually think this flows a bit better without the `it`:

Skaffold **will** do something, and when something happens, **will** do something else.